repose-aggregator/components/filters/derp/src/main/scala/org/openrepose/filters/derp/DerpFilter.scala

```diff
@@ -81,7 +81,7 @@ class DerpFilter extends Filter with HttpDelegationManager with LazyLogging {
     httpServletResponse.setContentLength(responseBody.length)
     httpServletResponse.setContentType(MediaType.TEXT_PLAIN)
     httpServletResponse.getWriter.write(responseBody)
-    httpServletResponse.sendError(statusCode)
+    httpServletResponse.sendError(statusCode, responseBody)
```

Comment:
If we fix sendError in the wrapper, we can get rid of the getWriter.write line again. That line is only used because using sendError on our wrapper is/was setting the status message rather than the body.